internal/jobservice/application.go

```diff
@@ -95,6 +82,12 @@ func (a *App) StartUp(ctx context.Context, config *configuration.JobServiceConfi
 
 	RectifyConfig(config)
 
+	if os.Getenv("JOBSERVICE_DEBUG") != "" {
+		log.SetLevel(log.DebugLevel)
+		log.Debug("Set logging to debug level")
+		log.Debugf("Subscription time: %d", config.SubscribeJobSetTime)
```

Comment:
This suggestion isn't prompted by your changes here, but reminded me of something that I've thought we should change for a while:  can we rename `JobServiceConfiguration.SubscribeJobSetTime`? It's an `int64` value that actually denotes the number of seconds before a job-set is considered expired, and having it named with a "Time" suffix implies that it's some sort of changing timestamp. What do you think about renaming it to `SubscriptionExpirySecs` or somesuch?  The current name has misled reviewers in other PRs into thinking it's a timestamp value that should get regular updates, when it actually does not.

Reply:
Fine by me. Better names make everything clearer.

Reply:
Addressed in https://github.com/armadaproject/armada/pull/2545/commits/4d6715a1443583284a1b2b36438544fbf65003b1